corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -63,16 +64,17 @@ def convert_to_date(date):
     return result
 
 
-def get_active_case_counts_by_owner(domain, datespan, case_types=None, owner_ids=None):
-    return _get_case_case_counts_by_owner(domain, datespan, case_types, False, owner_ids)
+def get_active_case_counts_by_owner(domain, datespan, case_types=None, owner_ids=None, export=False):
```

Comment:
Can we rename this to `use_export_instance` or pass the es_instance to be used?
If you pass the es_instance, this can be customised further to pass instance with timeout configured just for ICDS by adding it to some localsetting instead changing it for all environments?

Reply:
I passed it as a generic `export` flag under the assumption that down the line we may want to do other things different in export vs web ui requests, and this gives us that lever. Do you think restricting to only the instance makes more sense? I also figured this would be a positive change for all servers (so the exported reports dont time out).

Reply:
The PR description said "so icds can get some bigger reports", so i saw this more as ICDS specific change.

> export vs web ui requests
Didn't get that. What do you mean by this?

We always bank upon to export_instance for any longer searches on ES even if they are not exports. So i saw that we can break them down to be more specific cases for ES instances specific to what purpose they should be solving.

Reply:
> export vs web ui requests

This change only applies when you download a report as an excel file, not for normal views where it gets loaded in a web page. That is why I thought export was appropriate (its a report export) and what I meant by that statement.

Reply:
Oh i see what you mean. Sending export as True when you are serving exports.
Seems fine then. Should still consider renaming.

Reply:
maybe just `for_export`